pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -244,7 +244,7 @@ let
       is_clang = stdenv.cc.isClang;
       clang_use_chrome_plugins = false;
       blink_symbol_level = 0;
-      enable_swiftshader = false;
+      enable_swiftshader = true;
```

Comment:
Ideally I think it would be nice to remove this line to keep it as simple as possible (i.e. we only set values where we differ from the default), but that's optional. The upstream default can be seen [here](https://source.chromium.org/chromium/chromium/src/+/80.0.3987.149:ui/gl/BUILD.gn;l=15?originalUrl=https:%2F%2Fcs.chromium.org%2F).

Reply:
Good idea.

Done.